apps/web/src/views/AddLiquidityV3/formViews/V3FormView/components/FeeOption.tsx

```diff
@@ -20,7 +20,7 @@ const FeeOptionContainer = styled.div<{ active: boolean }>`
   border-radius: 16px;
   padding: 2px 2px 4px 2px;
   &:hover {
-    opacity: 0.7;
+    filter: brightness(1.5);
```

Comment:
what's the logic of the brightness value, it looks so dazzling
![image](https://github.com/user-attachments/assets/0284eafe-4743-4549-83f9-edf8788b3d62)